entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+# helm requested, sync the infoblox s3 bucket
+if [ -z "$NOAWS" ] && [ "$1" = 'helm' ] && [ "$2" != 'version' ] && [ "$2" != 'lint' ] && [ "$2" != 'template' ];
```

Comment:
`docker run -e NOAWS=1` is a way to use this container without requiring aws credentials. It also excludes a number of subcommands that do not use a repo server.